mypy.ini

```diff
@@ -51,6 +51,11 @@ disallow_untyped_defs = True
 warn_return_any = True
 no_implicit_optional = True
 
+[mypy-app.forms.fields.*]
```

Comment:
Since all of the forms folder is now types, you can remove all the `form.` rules and replace it with a single:

```ini
[mypy-app.forms.*]
disallow_untyped_defs = True
warn_return_any = True
no_implicit_optional = True
```

Reply:
Updated, thanks 👍